src/kaltura-player.ts

```diff
@@ -182,6 +182,9 @@ export class KalturaPlayer extends FakeEventTarget {
     const playerConfig = Utils.Object.copyDeep(mediaConfig);
     //merge the current sources from player to keep the sources passed from constructor earlier
     const sources = Utils.Object.mergeDeep({}, playerConfig.sources, this._localPlayer.sources);
+    if (playerConfig.sources.type === 'Vod' && this._localPlayer.sources.mediaEntryType === 'Live') {
```

Comment:
is there an enum to use for `Vod` and `Live` instead of hard-coded strings?
**Also**, can you please verify that this is not messing up https://github.com/kaltura/kaltura-player-js/pull/651 - when you switch to the live/vod entry in playlist, verify that the payload of `PLAYLIST_ITEM_CHANGED` contains `entryId` field with the "original" entry id (the live one).

Reply:
There is no enum, it's define as string.
Verified, it doesn't affect your patch, still send the live id.

Reply:
there's `MediaType` enum. see https://github.com/kaltura/playkit-js/blob/074433785174ea69440693b5d59e3fa11268cf6b/src/enums/media-type.ts#L1
i think it is better to use the enum, you can do something like:
`playerConfig.sources.type === this.MediaType.VOD......` since there's a getter for `MediaType`.

Reply:
done